message_sender/tasks.py

```diff
@@ -135,7 +142,22 @@ def run(self, message_id, **kwargs):
                         "metric_name": 'vumimessage.tries.sum',
                         "metric_value": 1.0
                     })
+
+                    # OBD number of successful tries metric
+                    if "voice_speech_url" in message.metadata:
+                        fire_metric.apply_async(kwargs={
+                            "metric_name": 'vumimessage.obd.successful.sum',
+                            "metric_value": 1.0
+                        })
```

Comment:
This will only apply to voice messages, not all messages. Also, firing here doesn't actually tell us that the message was successful. We can only know that the send was successful if we get an acknowledgement back, which we get here https://github.com/praekelt/seed-message-sender/blob/develop/message_sender/views.py#L79-L133 and here https://github.com/praekelt/seed-message-sender/blob/develop/message_sender/views.py#L144-L191